doc/tech/roadmap.md

```diff
@@ -46,7 +48,17 @@ A Release is:
 Please refer to individual [release plan](./release_notes/) and features for detailed roadmap.
 
 <a name="2.3"></a>
-### Milestones
+### Detailed Milestones
+| Review | Milestone | Description | Activities | Comments |
+|--------|-----------|--------------|------------|---------|
+| Kick-Off | M0 | The goal of the release Kick-Off is to open the innovation platform for the intent to participate in the CNTT release. Release Kick-Off review takes place for each releases. | Identity a list of features and functionality will be documented and developed. | |
+| Planning | M1 | The goal of the Release Planning review is to ensure the plan is complete, sufficient, and aligned with release milestones. All people resources are identified, documented and committed. | Identify what is in or out of scope. During the development phase, it helps reduce the discussion. Highlight the issues to TSC. | |
```

Comment:
Is it "Release Planning" or "Release Planning review"? If a review then where is the release plan developed?
Sentence issues

Reply:
After M0 and the day before M1 is the release planning. Review happens on the cut-off date as per the agreed M1 date to see what goes in and what goes out as per the delivery timeline in the release process. Hence I have defined release planning review.  M0 to M1 is Release Planning and M1 cutoff date is the day for Release Planning Review.